DESCRIPTION

```diff
@@ -34,7 +34,7 @@ BugReports: https://github.com/rstudio/reticulate/issues
 SystemRequirements: Python (>= 2.7.0)
 Encoding: UTF-8
 Depends:
-    R (>= 3.0)
+    R (>= 3.4)
```

Comment:
Or we could just use >= 3.5?

Reply:
I think this is the best option. Otherwise, we'll have to manually install older versions of rlang (and testthat!). 
Trying to run the current CI with R 3.4 I see this:
```
Error: 
  ! error in pak subprocess
  Caused by error: 
  ! Could not solve package dependencies:
  * deps::.: Can't install dependency rlang
  * rlang: Needs R >= 3.5.0
  * rcmdcheck: Can't install dependency webfakes
  * webfakes: Needs R >= 3.5.0
  * remotes: Can't install dependency webfakes
  * any::sessioninfo: dependency conflict
```

Reply:
Oof. Let's just bump to 3.5 then :-)